react-frontend/src/components/ArrowButton.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import arrow from "../assets/arrow.png";
+import "./ArrowButton.css"
+
+function ArrowButton() {
+  /* To handle the onClick() prop, make a div around this element and put the onClick() there.
```

Comment:
Interesting definitely not ai generated comment. but it is a good change to add to give the button functionality